sources/org.osbuild.files

```diff
@@ -93,7 +118,7 @@ def main(options, checksums, cache, output):
                 "--reflink=auto",
                 f"{cache}/{checksum}",
                 f"{output}/{checksum}"],
-                           check=True)
+                check=True)
```

Comment:
:eyes:
(Feel free to keep this fix included, but I did catch that!)

Reply:
Including this but in a separate commit. Thanks for the catch.